Common subdirectories: tmda-cgi-0.16.4/button_maker and tmda-cgi-0.16.4.fix/button_maker
Common subdirectories: tmda-cgi-0.16.4/display and tmda-cgi-0.16.4.fix/display
Common subdirectories: tmda-cgi-0.16.4/ht2html and tmda-cgi-0.16.4.fix/ht2html
Common subdirectories: tmda-cgi-0.16.4/htdocs and tmda-cgi-0.16.4.fix/htdocs
Common subdirectories: tmda-cgi-0.16.4/skel and tmda-cgi-0.16.4.fix/skel
Common subdirectories: tmda-cgi-0.16.4/stubs and tmda-cgi-0.16.4.fix/stubs
diff -up tmda-cgi-0.16.4/tmda-cgi.py tmda-cgi-0.16.4.fix/tmda-cgi.py
--- tmda-cgi-0.16.4/tmda-cgi.py	2003-12-15 08:12:32.000000000 -0800
+++ tmda-cgi-0.16.4.fix/tmda-cgi.py	2009-08-10 14:47:29.000000000 -0700
@@ -21,24 +21,6 @@
 
 "Web interface to TMDA tools."
 
-# Python2.3 seems to have a problem when Defaults.py imports various packages
-# from its directory.  This nasty hack intercepts those imports and adds a
-# "from TMDA" to them.
-OldImport = __builtins__.__import__
-def NewImport(name, globals=None, locals=None, fromlist=None):
-  if fromlist == None:
-    if name == "Errors":
-      from TMDA import Errors
-      return Errors
-    if name == "Util":
-      from TMDA import Util
-      return Util
-    if name == "Version":
-      from TMDA import Version
-      return Version
-  return OldImport(name, globals, locals, fromlist)
-__builtins__.__import__ = NewImport
-
 # Prepare the traceback in case of uncaught exception
 import os
 import sys
